test/parallel/test-assert.js

```diff
@@ -43,6 +43,7 @@ assert.ok(a.AssertionError.prototype instanceof Error,
 
 assert.throws(() => a(false), a.AssertionError, 'ok(false)');
 assert.throws(() => a.ok(false), a.AssertionError, 'ok(false)');
+assert.throws(() => a.ok(false, true, new Error('ok(false)')), a.AssertionError);
```

Comment:
`assert.ok()` takes up to two arguments, but here, there are three. Is that an error? If not, can you add a comment explaining what's going on?

Reply:
Sorry, my bad. I just updated the test